src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2071,19 +2069,21 @@ class CgroupUtils:
                 pbs.logmsg(pbs.EVENT_DEBUG2,
                            "%s: Job %s contains defunct process" %
                            (caller_name(), jobid))
-                raise CgroupProcessingError("Job contains defunct process")
+                # Use a list comprehension to remove all instances of the
+                # number 1
+                pids = [x for x in pids if x != 1]
         # check pids to make sure that they are owned by the job owner
         if pbs_version[0] >= 13:
```

Comment:
The cgroups hook that ships with 17 will work in a certain way.  The cgroups hook that ships in a later release will work in a new way.  The admin should use the hook provided with the release they have installed like with the rest of PBS works.  You don't suddenly replace qsub with a newer version.  Why should it work the same with the cgroups hook. 
 We shouldn't try and write one hook that runs everywhere.  It gets very ugly very fast.